webpack/components/EditableTextInput/EditableTextInput.js

```diff
@@ -86,6 +86,7 @@ const EditableTextInput = ({
       </SplitItem>
       <SplitItem>
         <Button
+          ouiaId={`submit-button ${attribute}`}
```

Comment:
```suggestion
          ouiaId={`submit-button-${attribute}`}
```
Feels a bit more consistent if we add the dash here so it's `edit-button-cdn-password` and not `edit-button cdn-password`

(applies to the buttons below as well)